Gemfile

```diff
@@ -27,4 +27,5 @@ group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.6"
   gem "jekyll-jupyter-notebook"
   gem "jekyll-seo-tag"
+  gem 'jekyll-redirect-from'
```

Comment:
I am not really familiar with Jekyll and thus I don't know if this is the right approach or not for creating a redirect 🤔  It seems to work but has lots of repeititon

Ideally I would like to just make a table with old URLs and redirects but I couldn't figure out how to do that